.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+# See https://pre-commit.com for more information
+repos:
+  - repo: https://github.com/PyCQA/flake8
+    rev: 6.1.0
+    hooks:
+      - id: flake8
+        # Required to make flake8 read from pyproject.toml for now :(
+        additional_dependencies: ["flake8-pyproject"]
+  - repo: https://github.com/psf/black
+    rev: 23.10.1
+    hooks:
+      - id: black
+  - repo: https://gitlab.com/vojko.pribudic/pre-commit-update
+    rev: v0.1.0
+    hooks:
+      - id: pre-commit-update
```

Comment:
This hook is outside of ticket scope. Also it was not in the previous reviewed PR
Could we remove it and analyze its implication when WAZO-3282 will be merged?